webdriverio/webdriverio.d.ts

```diff
@@ -0,0 +1,1057 @@
+// Type definitions for webdriverio 3.2.6
+// Project: http://www.webdriver.io/
+// Definitions by: Nick Malaguti <https://github.com/nmalaguti/>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference path="../node/node.d.ts"/>
+/// <reference path="../q/Q.d.ts"/>
+
+declare namespace WebdriverIO {
+    // EventEmitter
+    export interface IClient<T> {
```

Comment:
http://www.typescriptlang.org/Handbook#writing-dts-files

> Naming Conventions
In general, do not prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.